File: Swiper/Swiper.Server/Models/UserDTO.cs
namespace Swiper.Server.Models
{
    public class UserDTO
    {
        public string? Id { get; set; }
        public string? UserName { get; set; }
        public string? Email { get; set; }
        public List<ImageDTO>? Images { get; set; }
        public int? Age { get; set; }
        public string? Residence { get; set; }
    }
}

File: Swiper/swiper.client/src/generatedTypes/UserDTO.ts

import { ImageDTO } from "./ImageDTO";

export class UserDTO
{
    id: string | null;
    userName: string | null;
    email: string | null;
    images: ImageDTO[] | null;
    age: number | null;
    residence: string | null;

    constructor(id?: string | null,userName?: string | null,email?: string | null,images?: ImageDTO[] | null,age?: number | null,residence?: string | null,)
    {
    
        this.id = id ?? null;
        this.userName = userName ?? null;
        this.email = email ?? null;
        this.images = images ?? null;
        this.age = age ?? null;
        this.residence = residence ?? null;
    }
}